Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -3819,3 +4064,10 @@ openDateTimePicker state =
       _ <- launchAff $ showDateTimePicker push DateTimePickerAction
       pure NoAction
     ]
+
+
+truncate :: Int -> Maybe Number -> Maybe Number
```

Comment:
we can move these functions to mobilityPrelude

Reply:
It's specific to my use case